redvanworkshop/website
# RBD-6 - Linked In Follow

#### What's this PR do?

Adds a linked in follow us button at the bottom of the Careers page

#### Where should the reviewer start?

View the Careers page, scroll to the bottom

#### How should this be manually tested?

* Verify the button shows up
* Verify the button is in the right place
* Verify the button correctly goes to RVW's LinkedIn page

#### Any background context you want to provide?

n/a

#### What are the relevant github issue?

none

Jira:
https://redvanworkshop.atlassian.net/jira/software/projects/RBD/boards/21?assignee=5c0c0fb5128c7106f5768fab&selectedIssue=RBD-6

#### Screenshots (if appropriate)

n/a

#### What gif best describes this PR or how it makes you feel?

Quick QOL change

#### Definition of Done:

- [x] You have verified that either `npm test` or `npm run build` passes without issue
